Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift

```diff
@@ -61,7 +61,7 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        //TODO: Answers.logContentView(withName: "Product's detail", contentType: "product_detail", contentId: product.barcode, customAttributes: ["product_name": product.name ?? ""])
+        // Answers.logContentView(withName: "Product's detail", contentType: "product_detail", contentId: product.barcode, customAttributes: ["product_name": product.name ?? ""])
```

Comment:
This is not part of this current PR, but as it has been touched: Answers is part of Crashlytics, which is not used in the app anymore. I think we added the same behaviour using sentry breadcrumbs, we should check if we did and then remove this comment

Reply:
I always try to remove some of the swiftlint(?) warnings.

Reply:
But TODOs are probably here for a reason ?

Reply:
true